src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService_FindImplementations.cs

```diff
@@ -103,23 +103,60 @@ public async Task FindImplementationsAsync(Document document, int position, IFin
             var linkedSymbols = await SymbolFinder.FindLinkedSymbolsAsync(
                 symbol, solution, cancellationToken).ConfigureAwait(false);
 
+            // Because we're searching linked files, we may get many symbols that are conceptually 
```

Comment:
Are there tests that exist for this scenario today?

Reply:
I looked and could not find. I will add in follow-up. Thanks!